client/signup/steps/import-url/index.jsx

```diff
@@ -68,14 +68,18 @@ class ImportURLStepComponent extends Component {
 
 	handleSubmit = event => {
 		event.preventDefault();
-		const isValid = this.validateUrl();
+		const { flowName, stepName, translate, urlInputValue } = this.props;
+
+		this.props.recordTracksEvent( 'calypso_signup_import_url', {
+			flow: flowName,
+			url: urlInputValue,
+		} );
 
+		const isValid = this.validateUrl();
```

Comment:
Can we add isValid to the Tracks event? I think it'd be helpful for us to quickly see whether or not we should even bother further diving into this event.

Reply:
💯 -- including `isValid` will make this event much more valuable.

Reply:
Thanks, now added.